css.js

```diff
@@ -50,10 +64,17 @@ if (typeof window !== 'undefined') {
       } else {
         webkitLoadCheck(link, _callback);
       }
+      
       link.onerror = function(event) {
         _callback(event.error || new Error('Error loading CSS file.'));
       };
-      head.appendChild(link);
+
+      if (present)        
+         link.href = url + '?' + new Date().getTime();
```

Comment:
Because we aim to callback on load completion of the CSS to allow style queries, we should probably remove the existing CSS and re-inject rather?